executor_manager/executors/base.py

```
@@ -38,3 +41,86 @@ def get_executor_count(
+
+        Returns:
+            Dict with the following fields:
+                - exists (bool): Whether executor exists
+                - status (str): Executor status (running/exited/succeeded/failed/etc)
+                - oom_killed (bool): Whether executor was killed due to OOM
+                - exit_code (int): Exit code (0 = success, 137 = SIGKILL, etc)
+                - error_msg (str): Error message if any
+        """
+        pass
+
+    def get_executor_task_id(self, executor_name: str) -> Optional[str]:
```

Ruff (0.14.13)

Unused method argument: `executor_name`

(ARG002)